docs/en/operations/system-tables.md

```diff
@@ -20,7 +20,7 @@ System tables:
 
 Most of system tables store their data in RAM. ClickHouse server creates such system tables at the start.
 
-The [metric_log](#system_tables-metric_log), [query_log](#system_tables-query_log), [query_thread_log](#system_tables-query_thread_log), [trace_log](#system_tables-trace_log) system tables store data in a storage filesystem. You can alter them or remove from a disk manually. If you remove one of that tables from a disk, the ClickHouse server creates the table again at the time of the next recording. A storage period for these tables is not limited, and ClickHouse server doesn't delete their data automatically. You need to organize removing of outdated logs by yourself. For example, you can use [TTL](../sql-reference/statements/alter.md#manipulations-with-table-ttl) settings for removing outdated log records. 
+The system tables [metric_log](#system_tables-metric_log), [query_log](#system_tables-query_log), [query_thread_log](#system_tables-query_thread_log), [trace_log](#system_tables-trace_log) are served by [MergeTree](../engines/table-engines/mergetree-family/mergetree.md) table engine that store data in a storage filesystem. You can alter some properties of these tables or remove them from a filesystem manually. If you remove a table from a filesystem, the ClickHouse server creates the empty one again at the time of the next data writing. A storage period for these tables is not limited, and ClickHouse server doesn't delete their data automatically. You need to organize removing of outdated logs by yourself. For example, you can use [TTL](../sql-reference/statements/alter.md#manipulations-with-table-ttl) settings for removing outdated log records. Also you can use the partitioning feature of `MergeTree`-engine tables.
```

Comment:
`store data in a storage filesystem` - not clear, what we wanted to say.
`A storage period for these tables is not limited, and ClickHouse server doesn't delete their data automatically` - Better to tell that default behavior is not to remove this data, but we can use TTL.

`You can alter some properties of these tables or remove them from a filesystem manually.` it is not clear why we want to do so. lets check that properties are not changes via configuration.

If system table Schema changed in new release old tables are renamed and new ones are created.